pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -301,6 +317,31 @@ func (f *framework) RunFilterPlugins(pc *PluginContext,
 	return nil
 }
 
+// RunPostFilterPlugins runs the set of configured post-filter plugins.
+func (f *framework) RunPostFilterPlugins(
+	pc *PluginContext,
+	pod *v1.Pod,
+	nodes []*v1.Node,
+	failedFilters FailedFilterMap,
+) *Status {
+	for _, pl := range f.postFilterPlugins {
+		status := pl.PostFilter(pc, pod, nodes, failedFilters)
+		if status.IsSuccess() {
+			continue
+		}
+
+		if status.Code() == Unschedulable {
```

Comment:
Since `Post-filter` is an informational extension point, what does it mean if it returns a `Unschedulable` status? I think it might be enough to check whether it is `Success`.

Reply:
Sounds Good